rotkehlchen/exchanges/bitstamp.py

```diff
@@ -158,7 +165,7 @@ def query_balances(self) -> Tuple[Optional[Dict[Asset, Balance]], str]:
 
             symbol = entry.split('_')[0]  # If no `_`, defaults to entry
             try:
-                asset = Asset(symbol)
+                asset = asset_from_bitstamp(symbol)
```

Comment:
This can now also raise a `DeserializationErrror` from the function you implemented. You need to handle it.

It's good to be writting the exceptions a function can raise (also from subfunctions) in the docstring. This way you can sort of track what exceptions to expect where. Not perfect I know ... but it's something.